source/SH3/system/glcontext.cpp

```diff
@@ -17,11 +17,109 @@
 #include "SH3/system/glcontext.hpp"
 #include "SH3/system/log.hpp"
 #include "SH3/system/window.hpp"
+#include "SH3/glcallback.hpp"
 #include <GL/glew.h>
 #include <GL/glu.h>
 #include <GL/glext.h>
 #include <glm/gtc/matrix_transform.hpp>
 
+/**
+ *  OpenGL error callback function. Used in conjunction with glDebugMessageCallback
```

Comment:
End the documentation lines with a dot.

Add an empty line (just ` *  `) before the second sentence to make it part of only the long description.